diff --git a/index.js b/index.js
index b1a9e1a02e202baa4e2e0d5e81e14c5c9fd981b4..6952806d4a5b8c1ac950e0fc43027138f666c01a 100644
--- a/index.js
+++ b/index.js
@@ -1,10 +1,8 @@
-'use strict';
+import { PassThrough } from 'node:stream';
 
-const { PassThrough } = require('stream');
-
-module.exports = function (/*streams...*/) {
-  var sources = []
-  var output  = new PassThrough({objectMode: true})
+export default function mergeStream(/*streams...*/) {
+  let sources = []
+  let output  = new PassThrough({objectMode: true})
 
   output.setMaxListeners(0)
 
diff --git a/package.json b/package.json
index 1a4c54ca50905aadb1341705310169c222c0c6ac..ff0d15b07b454db7e393c3ff308ce3fab295bbdc 100644
--- a/package.json
+++ b/package.json
@@ -5,6 +5,8 @@
   "files": [
     "index.js"
   ],
+  "type": "module",
+  "main": "index.js",
   "scripts": {
     "test": "istanbul cover test.js && istanbul check-cover --statements 100 --branches 100"
   },
